config.example.wikimedia.yaml

```diff
@@ -148,6 +149,91 @@ templates:
                 request:
                   uri: http://graphoid.wikimedia.org/{domain}/v1/png/{title}/{revision}/{graph_id}
 
+      /{module:summary}:
+        x-subspec:
+          info: Summary sys module API
+          paths:
+            /request/{title}:
+              get:
+                x-setup-handler:
+                  - init:
+                      method: 'put'
+                      uri: /{domain}/sys/key_value/summary
+                      body:
+                        valueType: 'json'
+                        version: 1
+                x-request-handler:
+                  - get_revision:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/page_revisions/page/{title}
+                        headers:
+                          cache-control: '{cache-control}'
+                  - process_request:
+                      request:
+                        method: get
+                        uri: /{domain}/sys/summary/handler/{$$.default($.request.headers.cache-control, 'none-given')}
+                        body: '{$.get_revision.body.items[0]}'
+            /handler/no-cache:
+              get:
+                x-request-handler:
+                  - get_extracts:
+                      request:
+                        method: post
+                        uri: /{domain}/sys/action/query
+                        body:
+                          prop: 'extracts|pageimages'
+                          redirects: 'true'
+                          exsentences: 5
+                          explaintext: 'true'
+                          piprop: 'thumbnail'
+                          pithumbsize: 320
+                          titles: '{$.request.body.title}'
+                  - store:
+                      request:
+                        method: put
+                        uri: /{domain}/sys/key_value/summary/{$.request.body.title}
+                        headers:
+                          content-type: application/json
+                        body:
+                          title: '{$.get_extracts.body.items[0].title}'
+                          extract: '{$.get_extracts.body.items[0].extract}'
+                          thumbnail: '{$$.default($.get_extracts.body.items[0].thumbnail, null)}'
+                          timestamp: '{$.request.body.timestamp}'
+                  - return_result:
+                      return:
+                        status: 200
+                        headers:
+                          content-type: application/json
+                          etag: '{$.request.body.rev}/{$.request.body.tid}'
```

Comment:
We usually surround ETags with double quotes. But since key_value returns an ETag itself, it'd be good to use that one.